### DISCUSSION TITLE
deepseek中转api用不了吗？显示400 status code(no body)

### DISCUSSION THREAD

**Discussion by a1234tuan** [ZuodaoTech/everyone-can-use-english#713] [General]
我配置的接口地址是deepseek接口，加上/v1了，接口地址https://api.deepseek.com/v1，秘钥也配置了，用的deepseek秘钥，但是一直显示错误

**Comment by an-lee** [comment]
兼容 OpenAI 的应该都能用，报 400 估计是你接口地址配置错了吧
